src/elements/content-sidebar/versions/VersionsSidebar.scss

```diff
@@ -0,0 +1,8 @@
+.bcs-versions {
+    .bcs-scroll-content {
+        padding-left: 25px;
```

Comment:
`padding: 10px 25px 0 25px;` ?

Reply:
We should avoid setting style properties that we don't actually need (padding-bottom, in this case). It makes style inheritance and overrides more difficult when we constantly re-set values.